src/tools/default_flags_table/reco_flags.py

```diff
@@ -74,7 +74,100 @@
+    ('BEMC:EcalBarrelImagingRecHits:layerField',                'layer',                           ''),
+    ('BEMC:EcalBarrelImagingRecHits:sectorField',               'module',                          ''),
+    ('BEMC:EcalBarrelImagingRecHits:capacityADC',               '8096',                            ''),
+    ('BEMC:EcalBarrelImagingRecHits:pedestalMean',              '400',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:dynamicRangeADC',           '100*MeV',                         ''),
+    ('BEMC:EcalBarrelImagingRecHits:pedSigmaADC',               '3.2',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:thresholdADC',              '3.0',                             ''),
+    ('BEMC:EcalBarrelImagingRecHits:samplingFraction',          '0.10262666247845109',             ''),
+
+    ('BEMC:EcalBarrelScFiRecHits:input_tag',                    'EcalBarrelScFiRawHits',           'Name of input collection to use'),
+    ('BEMC:EcalBarrelScFiRecHits:capacityADC',                  '8096',                            ''),
+    ('BEMC:EcalBarrelScFiRecHits:dynamicRangeADC',              '100*MeV',                         ''),
+    ('BEMC:EcalBarrelScFiRecHits:pedestalMean',                 '400',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:pedestalSigma',                '3.2',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:resolutionTDC',                '10*picosecond',                   ''),
+    ('BEMC:EcalBarrelScFiRecHits:thresholdFactor',              '5.0',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:thresholdValue',               '0.0',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:samplingFraction',             '0.10262666247845109',             ''),
+
+    ('BEMC:EcalBarrelscFiMergedHits:input_tag',                 'EcalBarrelScFiRecHits',           ''),
+    ('BEMC:EcalBarrelscFiMergedHits:fields',                    'fiber',                           ''),
+    ('BEMC:EcalBarrelscFiMergedHits:refs',                      '1',                               ''),
+
+    ('BEMC:EcalBarrelImagingProtoClusters:input_tag',           'EcalBarrelImagingRecHits',        'Name of input collection to use'),
+    ('BEMC:EcalBarrelImagingProtoClusters::localDistXY',        '2.0*mm,2*mm',                     ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::layerDistEtaPhi',    '10*mrad,10*mrad',                 ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::sectorDist',         '3.0*cm',                          ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterHitEdep',  '0.',                              ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterCenterEdep', '0.',                            ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterEdep',     '0.5*MeV',                         ''),
```

Comment:
`minClusterNhits` should be 5 (don't see it listed)

Reply:
Added the minClusterNhits.

The 0.5/samplingFraction MeV I'm confused by. I think the RecHits have already divided out the sampling fraction and so are values of full energy deposition. Wouldn't this be double-applying it? Regardless, it seems there is no samplingFraction parameter available to use in the ImagingTopoClusterAlgorithm,

Reply:
The clustering algorithm variables had been tuned some time ago when the sampling fraction was applied only on the cluster level. I do not want to introduce the sampling fraction in the clustering, I'm just saying that the value of the parameter minClusterEdep should be 0.5/0.005 MeV

Reply:
So the min Cluster Energy = 0.5 MeV was appropriate when the clusters were constructed from the hits that were not corrected for the sampling fraction. It's just a constant scaling for this parameter.

Reply:
OK. I think I understand. So this means I should just set minClusterEdep to 100MeV since it is being applied to the full energy and not the sampled energy. Right?

Reply:
Exactly!